MicroWebSrv2/webRoute.py

```diff
@@ -54,7 +54,7 @@ def RegisterRoute(handler, method, routePath, name=None) :
                     if not argName :
                         raise Exception
                     argNames.append(argName)
-                    regex += '/([\\w.]*)'
+                    regex += '/([A-Za-z0-9_.]*)'
```

Comment:
A suggestion here. Can we add support for whitespace as well? In order to support links like
`http://example.com/some page` that would be translated from web browsers with a %20
I have tested in CPython to add _\\s_ and works but if micropython can't support _\\w_ I am not sure how to handle it